.github/workflows/build-images.yml

```diff
@@ -17,6 +17,7 @@
 #
 ---
 name: "Build Images"
+run-name: Build images for ${{ github.event.pull_request.title }}
```

Comment:
Would it be a good idea to also include the PR number?

Reply:
Possibly even URL to PR. I wonder how it will look like.